react_juce/core/CanvasView.cpp

```diff
@@ -611,6 +611,11 @@ namespace reactjuce
             if (!shouldAnimate && isTimerRunning())
                 stopTimer();
         }
+        
+        if (name == statefulProp)
```

Comment:
Kind of nit-picky but I think we might want `if (name == statefulProp && (bool) value)` It's possible, though unlikely, that someone would write `<Canvas stateful={false}>` in their js